BraintreeCore/src/main/java/com/braintreepayments/api/PaymentMethodNonce.kt

```diff
@@ -0,0 +1,47 @@
+package com.braintreepayments.api
+
+import android.os.Parcelable
+import android.os.Parcel
+import com.braintreepayments.api.PaymentMethodNonce
+
+/**
+ * Base class representing a method of payment for a customer. [PaymentMethodNonce] represents the
+ * common interface of all payment method nonces, and can be handled by a server interchangeably.
+ *
+ * @property string The nonce generated for this payment method by the Braintree gateway. The nonce will
+ * represent this PaymentMethod for the purposes of creating transactions and other monetary
+ * actions.
+ * @property isDefault `true` if this payment method is the default for the current customer, `false` otherwise
+ */
+open class PaymentMethodNonce : Parcelable {
+
+    companion object {
+        @JvmField
+        val CREATOR = object : Parcelable.Creator<PaymentMethodNonce> {
+            override fun createFromParcel(`in`: Parcel) = PaymentMethodNonce(`in`)
+            override fun newArray(size: Int) = arrayOfNulls<PaymentMethodNonce>(size)
+        }
+    }
+
+    open val string: String
```

Comment:
Yup! This isn't the best name lol. But we're stuck with it until the next major version 😮‍💨 .